common/defs/bitcoin/particl.json

```diff
@@ -0,0 +1,42 @@
+{
+  "coin_name": "Particl",
+  "coin_shortcut": "PART",
+  "coin_label": "Particl",
+  "website": "https://particl.io",
+  "github": "https://github.com/particl/particl-core",
+  "maintainer": "Ryno <ryno@particl.io>",
+  "curve_name": "secp256k1",
+  "address_type": 56,
+  "address_type_p2sh": 60,
+  "maxfee_kb": 2000000,
+  "minfee_kb": 1000,
+  "signed_message_header": "Bitcoin Signed Message:\n",
+  "hash_genesis_block": "0000ee0784c195317ac95623e22fddb8c7b8825dc3998e0bb924d66866eccf4c",
+  "xprv_magic": 1768850129,
+  "xpub_magic": 2401087160,
+  "xpub_magic_segwit_p2sh": 28471030,
+  "xpub_magic_segwit_native": 78792518,
+  "bech32_prefix": "bc",
+  "cashaddr_prefix": null,
+  "slip44": 44,
+  "segwit": true,
+  "decred": false,
+  "fork_id": null,
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Low": 10,
+    "Economy": 70,
+    "Normal": 140,
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 600,
```

Comment:
Shouldn’t this be 120 for Particl?

Reply:
correct

Reply:
My bad, I looked over it. It's not relevant for our particl-core integration though.
I will submit a patch once we add support to the official trezor wallet, which will require modifying this file again to add the blockbook and insight URLs.

I have not experienced any issues with transaction or message signing so far.